reference/configuration/framework.rst

```diff
@@ -515,7 +1169,7 @@ Whether or not to enable the ``translator`` service in the service container.
 fallback
 ........
 
-**default**: ``en``
+**type**: ``string`` **default**: ``'en'``
 
 This option is used when the translation key for the current locale wasn't found.
```

Comment:
but are possible (applies to all your comments). We usually use the short version in the docs as it tends to read a bit easier, a bit quicker.